lib/src/core/verifier/sample/mod.rs

```diff
@@ -0,0 +1,125 @@
+use anyhow::{anyhow, Context, Result};
+use std::path::PathBuf;
+extern crate serde;
+use self::serde::{Deserialize, Serialize};
+use super::*;
+use crate::core::verifier::policy::opa;
+use crate::fs::*;
+use async_trait::async_trait;
+use serde_json::{json, Value};
+
+mod policy;
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Ehd {
+    nonce: String,
+    public_key: String,
+}
+
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Tcb {
+    is_debuggable: bool,
+    cpusvn: u64,
+    svn: u64,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Quote {
+    is_debuggable: bool,
+    cpusvn: u64,
+    svn: u64,
+    report_data: String,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Policy {
+    allow: bool,
+    name: String,
+    hash: String,
+    diagnose: String,
```

Comment:
`name` -> `policy_name` It seems like all the policies have the same name anyway? 
`hash` -> `policy_hash` Not clear what this is the hash of otherwise.
`diagnose` -> `info`